cmd/plume/containerlinux.go

```diff
@@ -96,8 +97,8 @@ var (
 	}
 	specs = map[string]channelSpec{
 		"user": channelSpec{
-			BaseURL: "gs://users.developer.core-os.net/" + os.Getenv("USER") + "/boards",
-			Boards:  []string{"amd64-usr"},
+			BaseURL:       "gs://users.developer.core-os.net/" + os.Getenv("USER") + "/boards",
```

Comment:
how do we deal with the URLs which have `board` or `-usr` in them ? I assume some will go away because they are specific to CL?

Reply:
I'd expect that they all will.  Anything at `core-os.net` is for Container Linux.  In this particular case, all of `containerlinux.go` can (hopefully) be dropped.